lib/elixir/pages/anti-patterns/code-anti-patterns.md

```diff
@@ -319,7 +319,68 @@ This anti-pattern was formerly known as [Accessing non-existent Map/Struct field
 
 ## Dynamic atom creation
 
-TODO.
+#### Problem
+
+An `atom` is a primary data type of Elixir whose value is its own name. They are often useful to identify resources or express an operation's state. The creation of an `atom` do not characterize an anti-pattern by itself; however, `atoms` are not collected by Elixir's Garbage Collector, so values of this type live in memory while software is executing, during its entire lifetime. Also, BEAM limits the number of `atoms` that can exist in an application (`1_048_576`) and each `atom` has a maximum size limited to 255 Unicode code points. For these reasons, the dynamic atom creation is considered an anti-pattern, since in this way the developer has no control over how many `atoms` will be created during the software execution. This unpredictable scenario can expose the software to unexpected behavior caused by excessive memory usage, or even by reaching the maximum number of `atoms` possible.
```

Comment:
We shouldn't use backquotes for things that are not code, because it's not semantically right (and hinders accessibility as well 😢). Can you replace all the uses of `` `atom` `` and friends ("string" for example) with italic (`*atom*`) instead?

Reply:
Agreed. If we are going to use backquote, it should be `Atom`, but it should be used once or twice, and not on every reference.

Reply:
OK! I agree. I just made a commit to fix it. I'm sorry for that mistake... :)